ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/Config.scala

```diff
@@ -111,18 +113,21 @@ final class TrustStoreConfig private[sslconfig](
   def withData(data: Option[String]): TrustStoreConfig = copy(data = data, filePath = None)
   /** Disables `data` – only one of those can be used at any given time. */
   def withFilePath(filePath: Option[String]): TrustStoreConfig = copy(filePath = filePath, data = None)
+  def withPassword(password: Option[String]): TrustStoreConfig = copy(password = password)
   def withStoreType(value: String): TrustStoreConfig = copy(storeType = value)
 
   private def copy(
     data: Option[String] = data,
     filePath: Option[String] = filePath,
+    password: Option[String] = password,
     storeType: String = storeType): TrustStoreConfig = new TrustStoreConfig(
       data = data,
       filePath = filePath,
+      password = password,
       storeType = storeType)
 
   override def toString =
-    s"""TrustStoreConfig(${data},${filePath},${storeType})"""
+    s"""TrustStoreConfig(${data},${filePath},${password},${storeType})"""
```

Comment:
I included `password` because it was done so in `KeyStoreConfig` but actually I think we should not do that.